.dockerignore

```diff
@@ -1,5 +1,5 @@
-*
+
 !target/*-runner
 !target/*-runner.jar
 !target/lib/*
-!target/quarkus-app/*
\ No newline at end of file
+!target/quarkus-app/*
```

Comment:
If this directory exists, I believe we need to use this for the app and not the runner files. 
I think the cut-over was in Quarkus 1.12